opnode/rollup/driver/step.go

```diff
@@ -279,6 +286,28 @@ func attributesMatchBlock(attrs *l2.PayloadAttributes, parentHash common.Hash, b
 	}
 	for i, otx := range attrs.Transactions {
 		if expect := block.Transactions[i]; !bytes.Equal(otx, expect) {
+			//if otx[0] == types.DepositTxType && expect[0] == types.DepositTxType {
```

Comment:
Did you mean to leave this dead code in?

Reply:
thanks for catching that, that's some debug code, will remove

Reply:
removed it, please have a look again.